### ISSUE TITLE
Connection to Lavalink Nove "ws://localhost:8080" failed!

### ISSUE THREAD

**Issue by SonOfDagon** [angelobreuer/Lavalink4NET#50]
I realise that this is most likely a very noob question, but I'm unsure where else to ask this, so I'm sorry for submitting it here. Still, how do I go about solving this?

This is the output of a project I pretty much built using the ExampleBot from the samples directory.

![image](https://user-images.githubusercontent.com/54398437/91683504-17aa3680-eb55-11ea-8c84-f51f0fc34ea8.png)

**Comment by angelobreuer** [comment]
No need to apologize here.

It seems like your Lavalink server/node is not running or seems to have a different port or address.

The error says that no connection could be made to the node.

Thanks in advance!

**Comment by SonOfDagon** [comment]
> It seems like your Lavalink server/node is not running or seems to have a different port or address.

Having absolutely no experience with Java and networking (it's a miracle I've managed to get my bot to be hosted on a remote server in the first place), I somehow COMPLETELY missed the part where I needed to get the Lavalink server up - so I was merrily executing my code without ever having thought about executing Lavalink.jar.

Updating my Java JRE to version 13, downloading the binary release, creating application,yml, and running it with java -jar Lavalink.jar, it immediately worked. IMMEDIATELY. No issues.

You have no idea how much relief seeing the bot work has brought me. I was just sat there, listening to the music being played by the bot, and laughing. 

Consider the issue closed - and thank you for pointing out the obvious!
